internal/models/api/common.go

```
@@ -0,0 +1,10 @@
+package api
+
+type ValidationError struct {
+    Fields []string
```

Assuming we will always fail fast at the first validation error, lets keep this as `string` instead of `[]string`?

---

Ok will make the change